funcbench/env.go

```diff
@@ -157,7 +185,12 @@ func (g *GitHub) PostResults(tables []*benchstat.Table, extraInfo ...string) err
 		return err
 	}
 
-	legend := fmt.Sprintf("Old: `%s`\nNew: `PR-%d`", g.compareTarget, g.client.prNumber)
+	legend := fmt.Sprintf("Old: `%s`/`%s`\nNew: `PR-%d`/`%s`",
```

Comment:
I would say always use %v, this way if you change the env type don't have to think about updating the message. Fewer things to think about.